### ISSUE TITLE
Library jna missing

### ISSUE THREAD

**Issue by TimoScheuer** [payara/Payara#4023]
# Description #
----------

I set up a PAM realm via class com.sun.enterprise.security.auth.realm.pam.PAMRealm. The necessary library jna (I used version jna-4.2.2 from Ubuntu 16.04) is missing. Adding it via "add-library" does not work (all three types "common", "app" and "ext"). I had to copy this library to "payara5/glassfish/modules/ to make PAM authentication work.

## Expected Outcome

Either no additional library should be necessary or it should be possbile to add the library e.g. via "add-library".

## Current Outcome

  WEB9102: Web Login Failed: com.sun.enterprise.security.auth.login.common.LoginException: Login failed: java.lang.NoClassDefFoundError: com/sun/jna/Callback

## Steps to reproduce (Only for bug reports) 
Add PAM authentication Realm. And try to use it for authentication.

## Environment ##

- **Payara Version**: 5.191
- **Edition**:  Full
- **JDK Version**: OpenJDK 1.8.0_212
- **Operating System**: Ubuntu 16.04

**Comment by smillidge** [comment]
I believe the correct version currently referenced from Pam4j is 4.5.2

**Comment by TimoScheuer** [comment]
Meanwhile I switched to the current payara version 5.192. After more deep investigation I found out that everything necessary is there. Authentication was not working with the account I tried, but another account which I tried later on was working. The source of the problem is my PAM configuration which uses LDAP and Kerberos.

I think with the current findings the ticket can be closed or even removed because it does not provide any additional value for someone else;-(
